tests/yam/agama/import_agama_profile.pm

```diff
@@ -7,22 +7,26 @@
 use base Yam::Agama::patch_agama_base;
 use strict;
 use warnings;
-use testapi qw(assert_script_run data_url get_required_var select_console script_run);
+use testapi qw(assert_script_run data_url get_required_var set_var get_var select_console script_run);
 use autoyast qw(expand_agama_profile generate_json_profile);
 
 sub run {
     my $profile = get_required_var('AGAMA_PROFILE');
     my $profile_url = ($profile =~ /\.libsonnet/) ?
       generate_json_profile($profile) :
       expand_agama_profile($profile);
+    set_var('AGAMA_PROFILE', $profile_url);
 
     select_console 'install-shell';
 
-    # Workaround to import profile in each Agama version
-    my $command = script_run('agama config load --help | grep URL_OR_PATH') == '0' ?
-      "agama config load $profile_url" : "agama profile import $profile_url";
+    if (get_var('AGAMA_PROFILE_LOAD') eq undef || get_var('AGAMA_PROFILE_LOAD') == '1') {
+        # Workaround to import profile in each Agama version
```

Comment:
the workaround is not needed anymore, could you drop it so we can have more clarity here.

Reply:
this is not related to this ticket but ok.